lib/config/virtual_container_host.go

```diff
@@ -228,7 +233,7 @@ func (t *VirtualContainerHostConfigSpec) SetIsCreating(creating bool) {
 
 // IsCreating is checking if this configuration is for one creating VCH VM
 func (t *VirtualContainerHostConfigSpec) IsCreating() bool {
-	return t.ExecutorConfig.ID == CreatingVCH
+	return strings.HasPrefix(t.ExecutorConfig.ID, CreatingVCH)
```

Comment:
I better name CreatingVCH as CreatingVCHPrefix or something like that. I also would use "creating" instead of "CreatingVCH" to be consistent with "init"